share/completions/sad.fish

```diff
@@ -0,0 +1,9 @@
+complete sad    -s 0 -l read0                    -d 'Use \0 as stdin delimiter'
+complete sad    -s e -l exact                    -d 'String literal mode'
+complete sad -x -s f -l flags                    -d "Regex flags: use `--help` instead of `-h` to see details"
```

Comment:
What does this mean?

Does this treat `sad -h` different from `sad --help`? Is the `-h` version short help?

That's... unorthodox.

Reply:
Yes, exactly.

Reply:
Okay, in that case `-h` and `--help` are separate options, and they should then be separate to complete.

`-h` is "short help" and `--help` is "long help" or similar.

Reply:
I guess that makes sense, done.